src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.FSharp.xaml

```diff
@@ -33,6 +33,26 @@
                 HelpUrl="https://docs.microsoft.com/dotnet/fsharp/language-reference/compiler-options"
                 Category="General" />
 
+  <EnumProperty Name="FSharpPreferNetFrameworkTools"
```

Comment:
How do we know what the default value is?
Is it the first listed, last or is there another property?

The default should be:
- FSharpPreferNetFrameworkTools=false
- FSharpPrefer64BitTools=true

Reply:
By default, these properties are not written to the project file, if they aren't there.

Reply:
I'm going to check it on the latest dev17

Reply:
At the moment we're largely dependent on the imported .props and .targets (e.g. from the SDK) providing a default value which is then what we will show in the UI. If  the value isn't specified *anywhere* in the project or imports the property will have no value.

Reply:
Yeah, we discussed it internally, and will add a default value for it (as oppose to now it just having empty value, which is treated as default).